### ISSUE TITLE
tipuesearch plugin works with Lunr.js

### ISSUE THREAD

**Issue by mpaglia0** [pelican-plugins/tipue-search#7]
Hi all,
since the index produced by your plugin works perfectly with Lunr.js I think it will be useful to inform users. Tipuesearch is a defunct project while Lunr is still growing and improving. Moreover with Lunr you can get rid of jQuery since it has any external dependency. As regards speed I think it is very similar to tipuesearch. Elegant theme moved to Lunr.js from Tipuesearch and I have also run tests on my theme under development and now have Lunr search up and running without much efforts.

**Comment by justinmayer** [comment]
Hi @mpaglia0. I have already written a replacement for this plugin (that uses neither Tipue Search nor Lunr), and I will publish it here as soon as I can. I think you will find it to be a very useful option. 💫

**Comment by mpaglia0** [comment]
Oooops... thank you indeed!

**Comment by justinmayer** [comment]
Hi @mpaglia0! The replacement I mentioned above has now been released. Thank you for your patience. Following are the details…

I have created a new [Pelican Search](https://github.com/pelican-plugins/search) plugin. I highly suggest using that instead of Tipue Search.

This plugin and the repository are deprecated. The code is archived in the [Legacy Plugins Repository](https://github.com/getpelican/pelican-plugins/tree/master/tipue_search). This repository will eventually be deleted.
